server/game/cards/04-MM/ConsulPrimus.js

```diff
@@ -0,0 +1,29 @@
+const Card = require('../../Card.js');
+
+class ConsulPrimus extends Card {
+    setupCardAbilities(ability) {
+        this.reap({
+            condition: context => context.game.creaturesInPlay.length > 1 && context.game.creaturesInPlay.some(card => card.hasToken('amber')),
+            target: {
+                cardType: 'creature',
+                cardCondition: card => card.hasToken('amber'),
+                gameAction: ability.actions.removeAmber()
```

Comment:
This feels like it should be a "moveAmber" action and done all in one

Reply:
I do not think we have a moveAmber action at the moment. I think all cards that move amber does remove + place.

Reply:
Fancy adding one?

Reply:
It would require a double target :) I can try as well

Reply:
I did not implement the moveAmber. It has many implications which the engine cannot handle. I could implement a naive solution, but I could foresee cases where it would be difficult to adjust in the future. 

First is the problem of duplicating the same event type in multiple places. We have an addToken which could be modified to consider future card effects, which would have to be duplicated in MoveAmber.

A pure promptForSelect would not be able to apply all the canAffect/legalTarget validation which is part of the GameAction implementation. For example, if in the future we have a card that says "this creature cannot hold any amber", we could easily add an effect check for AddToken. We would have to do the same in MoveAmber pfs.

Ideally, the best thing to do would be some sort of composing gameactions allowing a multiple target as input.

Something like this:
```javascript
    condition: context => context.game.creaturesInPlay.length > 1,
            target: 
                source: {
                    cardType: 'creature',
                    cardCondition: card => card.hasToken('amber')                    
                },
                destination: {
                    //dependsOn: 'source'; implicit
                    cardType: 'creature'
                    // implicit cardCondition: card => card !== preContext.target 
                }                
            }),
            gameAction: ..moveAmber()
```

But I do not know how to do this double targe moveAmber.